explainaboard/analysis/performance.py

```diff
@@ -43,8 +45,13 @@ def dict_conv(cls, k: str, v: Any) -> Any:
         Returns:
             The value corresponding to the key
         """
+        serializer = PrimitiveSerializer()
+
         if k == 'performances':
```

Comment:
Should this be `results` instead of `performances`

Reply:
it seems that this has been fixed in: https://github.com/neulab/ExplainaBoard/pull/539/commits/34b0a6861adf054e9e7f68412ed104aa18626bb3